app/src/renderer/vuex/modules/delegates.js

```diff
@@ -69,28 +70,40 @@ export default ({ node }) => {
     },
     async getDelegates({ commit, dispatch }) {
```

Comment:
where are you setting `loading` to `true`?

Reply:
one line below your comment ;)